### DISCUSSION TITLE
Vertical Accuracy Reporting

### DISCUSSION THREAD

**Discussion by DrawQuick** [opengisch/QField#3890] [Bugs]
Hello Qgis Developers
I'm using a U-Blox c099-f9p evaluation board and connecting to an Ntrip on RTK2GO using Android App Bluetooth GNSS version 1.0.39.
This app reports the horizontal and vertical accuracy from the board correctly. Qfield shows the same horizontal accuracy but not the vertical accuracy. Typically a difference of +1.2metres. As an example, if the board is reporting 0.04m Qfield is reporting 1.24m.
Also under Settings, Positioning tab, 
"Bad accuracy below[m]" should read
"Bad accuracy above [m]"
and
"Excellent accuracy above[m]" should read
"Excellent accuracy below[m]"
Example: If the accuracy is above 1m that is bad accuracy and an accuracy below 0.05m is excellent.
Thank you

**Comment by Dr-Digitiser** [comment]
I agree with DrawQuick RE the wording for bad and excellent accuracy in the active accuracy indicator under position settings. The wording as stated is counter to (the opposite of) how the industry refers to these concepts.

**Comment by ConJaySten** [comment]
Did you ever resolve the vertical accuracy issue? The same thing is happening to me with GNSS Master and Qfield. GNSS Master shows 10-15mm vertical accuracy after NTRIP correction, but Qfield shows over a meter. I'm assuming it's some kind of problem with NMEA codes.

**Comment by DrawQuick** [comment]
Hi ConJaySten,
No, I haven't resolved the issue. I don't think this issue has been
assigned to any developer? I did notice that Qfield is erratic when
displaying the vertical. When the system is showing rtkfix with xy less
than 2cm, the z value is showing in the metre range or not available. I've
been trying to setup the Qfield development environment using QT in Windows
without success. Seems to be very complicated. I'm going to try QT in Linux
and see if I can actually get the app to compile.

**Reply by ConJaySten** [reply]
So I'm discussing a different issue with the developers, and I decided to uninstall Qfield and reinstall it.

I've had Qfield on my phone for _years_.  And I noticed _a long time ago_ they changed the file structure.  But I just kept importing my project from the old proj file directory created when I first installed Qfield

I deleted that directory after the reinstall and started using the "new" proj folder inside "Android\data\ch.opengis.qfield\files\QField\proj" and the vertical accuracy now reports correctly.
